Gemfile

```diff
@@ -55,6 +55,10 @@ group :development, :test do
   # Adds support for Capybara system testing and selenium driver
   gem 'capybara', '>= 2.15'
   gem 'selenium-webdriver'
+  gem 'database_cleaner'
```

Comment:
Do you config database cleaner?

Reply:
i didnt, shoul i? or just remove the gem  ?

Reply:
configured.